src/components/gb-card-4 - Copy.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {ClockSVG} from './svgs/ClockSVG';
+import {SourceSVG} from './svgs/SourceSVG';
+
+import '../style/gb-style.css';
+
+export const CardFour = () => {
+  return (
+     
+      <div className='component-to-copy gb-card-four-wrapper'>
+      <ul className='gb-card-four-sub-cards'>
+        <li className='gb-card-four-sub-card'>
```

Comment:
You should make an own component CardFoutLi, like we did at CardOne (and put the <li> elements in there)